tutorials/ab3AddRemark.md

```diff
@@ -10,7 +10,11 @@ Let's walk you through the implementation of a new command — `remark`.
 
 This command allows users of the AddressBook application to add optional remarks to people in their address book and edit it if required. The command should have the following format:
 
-`remark INDEX r/REMARK` (e.g., `remark 2 r/Likes baseball`)
+`remark INDEX r/REMARK`
+* `INDEX` refers to the list position of the person you want to add a remark to.
```

Comment:
As the list could be filtered, would this explanation be better? Perhaps we could phrase it as:
`Index` refers to the list position of the person in the currently displayed person list.

Reply:
Thanks for your feedback! 
I would revise it to simplify the language and also clarify that the position is relative to the current state of the list, whether filtered or not.